services/ingestion/providers/opendota/adapters.py

```
@@ -0,0 +1,29 @@
+#services/ingestion/providers/opendota/adapters.py
+from typing import Any, Dict
```

Ruff (0.15.2)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)